api/v1alpha1/entandoappv2_types.go

```diff
@@ -29,6 +29,8 @@ type EntandoAppV2Spec struct {
 	Version string `json:"version"`
 	// EntandoAppName is the name of the EntandoApp CR to update
 	EntandoAppName string `json:"entandoAppName,omitempty"`
+	// ImageSetType is to select the list of image to use, the admitted values are: RedhatCertified or Community
```

Comment:
is **used** to select the list of image**s**

Reply:
done